docs/rules/accessor-pairs.md

```diff
@@ -27,7 +28,7 @@ This rule warns if setters are defined without getters. Using an option `getWith
 
 ## Rule Details
 
-This rule enforces a style where it requires to have a getter for every object which has a setter defined. By activating the below option it enforces vice-versa behaviour also.
+This rule enforces a style where it requires to have a getter for every object which has a setter defined. By activating the option below it enforces vice-versa behaviour also.
```

Comment:
I think this should be changed a bit more. This paragraph is referring to the option below, but there are two options listed for this rule. It might be confusing for a user which option should enable opposite behavior.